pkg/operator/k8sutil/node.go

```diff
@@ -31,9 +31,14 @@ import (
 	helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
 )
 
+const (
+	TopologyLabelPrefix = "topology.rook.io/"
```

Comment:
It might not matter, but do you want the slash here?

Reply:
It was intentional, although it does possibly look awkward. It made parsing a bit simpler so I wouldn't have to put something else in place to prevent `topology.rook.io.blah/rack`

Reply:
Good enough, LGTM.